scipy/stats/tests/test_distributions.py

```diff
@@ -3263,6 +3263,68 @@ def test_entropy(self, mu, ref):
         assert_allclose(stats.invgauss.entropy(mu), ref, rtol=5e-14)
 
 
+class TestLandau:
+    @pytest.mark.fail_slow(60)
```

Comment:
These tests are quite slow. I don't think we need to test at multiple location parameters, and we don't need to test at 999 points. One thought is to leave them as they are, mark them `xslow`, and then just add a much faster test for a few points at `loc=0`. Does something like that sound good?

Reply:
Fully agree, One point to compare that we actually implement the correct distribution is enough.